ssr/src/state/admin_canisters.rs

```diff
@@ -4,8 +4,7 @@ use leptos::expect_context;
 use yral_canisters_client::{
     individual_user_template::IndividualUserTemplate, sns_swap::SnsSwap, user_index::UserIndex,
 };
-
-use crate::utils::ic::AgentWrapper;
+use yral_canisters_common::agent_wrapper::AgentWrapper;
 
 #[derive(Clone)]
 pub struct AdminCanisters {
```

Comment:
If possible might need to refactor AdminCanisters too because in the future the token swapping mechanism requires the admin canister to update the `last_swapped_price` by hitting the token creator's individual user canister.

Reply:
AdminCanisters is only available server side. We will expose the required REST endpoints to the mobile app, but the struct and the secret key wont go outside the ssr server